Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift

```diff
@@ -86,6 +86,11 @@ extension PhotonActionSheetProtocol {
                 Tab.ChangeUserAgent.updateDomainList(forUrl: url, isChangedUA: tab.changedUserAgent, isPrivate: tab.isPrivate)
             }
         }.items
+		
+		// Hard Refresh
+		let toggleHardRefresh = SingleActionViewModel(title: .AppMenu.AppMenuReloadWithoutCacheTitleString, iconString: ImageIdentifiers.reloadWithoutCache) { _ in
+			tab.reload(bypassCache: true)
+		}.items
```

Comment:
It was discussed in the task comments to only add a keyboard shortcut and not a menu option for this. Please remove the menu action and replace by a keyboard shortcut in the MenuBuilderHelper under the `viewMenu`. You can then remove the image identifiers and icon.

Reply:
Apologies, I read the initial issue and must have missed some of the comments. I have removed this and added a keyboard shortcut as requested. Please review latest commit.